align/pnr/checker.py

```diff
@@ -17,11 +17,11 @@ def check_placement(placement_verilog_d, scale_factor):
         with types.set_context(constraints):
             constraints.append(
                 constraint.SetBoundingBox(
-                    instance=module['abstract_name'],
-                    llx=bbox.llx//scale_factor,
-                    lly=bbox.lly//scale_factor,
-                    urx=bbox.urx//scale_factor,
-                    ury=bbox.ury//scale_factor,
+                    instance=module['concrete_name'],
```

Comment:
Made this a floating point divide in case a boundary coord can not be expressed in nanometers.

Reply:
This should not make a difference as checker is using integer variables right now.